unfoldingWord/translationCore
# Added Export To CSV Feature

#### This pull request addresses:

Write a few sentences about what your pull request addresses, the relevant issue numbers, etc.

This PR allows a user to see the checking data, comments, selections, verse edits, and reminders in csv form.

#### How to test this pull request:

Press the export button on any project. Select location and verify that it output correctly.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1471)
